web/src/admin/events/EventListPage.ts

```diff
@@ -82,9 +83,9 @@ export class EventListPage extends TablePage<Event> {
             html`<div>${actionToLabel(item.action)}</div>
                 <small>${item.app}</small>`,
             EventUser(item),
-            html`<span>${item.created?.toLocaleString()}</span>`,
+            html`<div>${getRelativeTime(item.created)}</div>
```

Comment:
Nifty.  I tend to reverse these: "Dec 10, 2023 (23 days ago)", but I think the human brain appreciates having a "distance from here / distance from now" along with a location or date.

Reply:
My brain is quite happy to get that relative info.